frontend/src/index.scss

```diff
@@ -200,3 +200,7 @@ p {
 .button-cell-border {
   border-bottom: 0 !important;
 }
+
+.usa-table thead th[aria-sort] {
```

Comment:
Why this is needed?

Reply:
To overwrite the original that is mislead. It is using an style we dont want to.